### PR TITLE
D3ASIM-2538: Enhancing testing scripts for aggregators to list market…

### PR DESCRIPTION
**Pull request by faizan2590**
… stats of selected devices.

### CODE REVIEW

File: tests/test_redis_aggregator.py [hunk 1]
```diff
@@ -74,5 +75,11 @@ def on_batch_response(self, market_stats):
 selected = pv.select_aggregator(aggregator.aggregator_uuid)
 logging.info(f"SELECTED: {selected}")
 
+
+redis_market = RedisMarketClient('house-2')
+market_slot_string_2 = today().add(minutes=60).format(DATE_TIME_FORMAT)
```
Comment (hannesdiedrich) [inline tests/test_redis_aggregator.py:80]: why `_2` ?

File: tests/test_create_aggregator.py [hunk 2]
```diff
@@ -106,6 +109,14 @@ def on_finish(self, finish_info):
 load2.select_aggregator(aggr.aggregator_uuid)
 pv1.select_aggregator(aggr.aggregator_uuid)
 
+area_uuid = get_area_uuid_from_area_name_and_collaboration_id(
+    simulation_id, "House", domain_name)
+
+rest_market = RestMarketClient(simulation_id, area_uuid, domain_name, websocket_domain_name)
+market_slot_string_2 = today().add(minutes=60).format(DATE_TIME_FORMAT)
+
 from time import sleep
```
Comment (hannesdiedrich) [inline tests/test_create_aggregator.py:118]: please move to top (saw it before, but forgot to comment). Thanks

File: tests/test_redis_aggregator.py (hunk 1 above)
Comment (faizan2590) [reply tests/test_redis_aggregator.py:80]: Just quick variable naming and copy/pasting. Will update with better name.

File: tests/test_redis_aggregator.py [hunk 3]
```diff
@@ -74,5 +75,11 @@ def on_batch_response(self, market_stats):
...
+
 while not aggregator.is_finished:
-    sleep(0.5)
+    list_market_stats_results = redis_market.list_market_stats([market_slot_string_2])
```
Comment (hannesdiedrich) [inline tests/test_redis_aggregator.py:83]: As discussed offline, this is just as reference for the aggregator user. Please move it to the on_market_cycle event and leave the while loop clean (and also revert the update time to 0.5 seconds). Thanks!

**Pull request merged**